lib/kanta_web/live/translations/translations_live/translations_live.ex

```diff
@@ -151,10 +160,18 @@ defmodule KantaWeb.Translations.TranslationsLive do
 
   defp get_not_translated_default_value(_), do: false
 
-  defp parse_page(page) do
-    case Integer.parse(page) do
-      {page, _} -> page
-      _ -> 1
-    end
+  defp parse_filters(filters) do
+    Enum.reduce(filters, %{}, fn {key, value}, acc ->
+      case key do
+        filter_key when filter_key in ["context_id", "domain_id", "locale_id"] ->
+          case parse_id_filter(value) do
+            {:ok, id} -> Map.put(acc, filter_key, id)
+            _ -> acc
+          end
+
+        filter_key ->
+          Map.put(acc, filter_key, value)
+      end
+    end)
```

Comment:
Maybe move some of the logic to a separate function to improve readability

Reply:
https://github.com/curiosum-dev/kanta/pull/52/commits/2ffe76466384c07af5217f6afafee00aec43f774